numpy/polynomial/chebyshev.py

```diff
@@ -1647,6 +1647,129 @@ def chebfit(x, y, deg, rcond=None, full=False, w=None):
     return pu._fit(chebvander, x, y, deg, rcond, full, w)
 
 
+def chebfit2d(x, y, z, deg, rcond=None, full=False, w=None, max_degree=None):
+    """
+    2d Least squares fit of Chebyshev series to data.
```

Comment:
I wonder if we should construct this docstring from a template, it irritates me to see such a large docstring for such a small function body.